src/python/pants/backend/terraform/goals/tailor.py

```diff
@@ -42,7 +42,7 @@ async def find_putative_terrform_module_targets(
         PutativeTarget.for_target_type(
             TerraformModuleTarget,
             path=dirname,
-            name=None,
+            name="tf",
```

Comment:
I think `tf` is too abbreviated. In the BUILD file, it won't be a problem, as the target alias will be descriptive there and give context, but looking at the address out of that context, I think it could become confusing.